src/fragments/MemberList.tsx

```diff
@@ -20,9 +20,11 @@ const MemberList: React.FC = () => {
       return;
     }
     memberListGetter(channel).then((resolved) => {
-      setChannelMembers(resolved);
+      if (resolved != channelMembers) {
```

Comment:
This check isn't needed, React itself checks if state changes before re-rendering